### PR TITLE
Keep css modules last

### PR DESCRIPTION
**Pull request by keepitterron**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -77,6 +77,8 @@ const style = ({
 
   const isResolvedComponent = imported => imported.moduleName.startsWith('components/');
 
+  const isCssModule = imported => imported.moduleName.startsWith('./styles.module');
```
Comment (vhmth) [inline src/index.js]: Should we make it mandatory that all style imports are last? Basically `/\.(css|less)$/.test(imported)`?

Comment (keepitterron) [reply src/index.js]: Good point. Actually we could push down any non-javascript imports.

Ideally though, I want no mandatory order for my css, cascading is crucial, I want to be able to decide which resource to load first.
I didn't try it, maybe it's possible to keep the order the user decide?

Comment (vhmth) [reply src/index.js]: True true. I’d be happy with keeping the order the user decides as long as the grouping is at the end. Maybe we:

1. Filter out styles from other imports.
2. Pick out styles imports and add to array *in order*
3. Spread that styles array last

**Comment by keepitterron** [comment]
Ok, it pushes styles to the bottom, keeping css modules last. And it keeps the user defined order

**Comment by johnnyopao** [comment]
Recent changes look good!

**Comment by keepitterron** [comment]
mmm...
how do we publish these changes?

**Comment by johnnyopao** [comment]
I think @paulius005 needs to publish or add you as a collaborator on npm

https://www.npmjs.com/package/import-sort-style-loom

File: src/index.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ const style = ({
 
   const isResolvedComponent = imported => imported.moduleName.startsWith('components/');
 
-  const isCssModule = imported => imported.moduleName.startsWith('./styles.module');
+  const isCssFile = imported => imported.moduleName.match(/(\.css|\.less)$/);
```
Comment (vhmth) [inline src/index.js]: yee yee

**Comment by vhmth** [comment]
@johnnyopao did you end up publishing `loom-embed` via your personal NPM account too?

**Comment by johnnyopao** [comment]
Looks like I made a loomhq org but didn't add anyone else to that 🤦🏻‍♂️

I'll ping everyone for their npm usernames to add them in

**Pull request merged**